### ISSUE TITLE
Fix Social Media Links Not Having Content

### ISSUE THREAD

**Issue by vkoves** [vkoves/vkoves.github.io#5]
A failure from HTML Codesniffer indicates that the social media links do not have any content (and thus no accessible name):

![screenshot from 2018-12-09 12-59-15](https://user-images.githubusercontent.com/3187531/49701485-9c4f6700-fbb2-11e8-8477-f9aa8930fe32.png)

Setting an `aria-label` should resolve this issue.
